### PR TITLE
Updating latest translations.

### PR DESCRIPTION
**Pull request by sivanatase1-ctxs**
updating available translations,

### CODE REVIEW

File: bundles/http/Citrix/66cf6f1a-e71d-44df-88c5-3db3e99ef7da/i18n/de.json [hunk 1]
```diff
@@ -1,148 +1,148 @@
...
-    "db7cc4ca-a6ff-4132-bb8e-bc13886dfcc2" : "Ort",
-    "59593fb3-5cd9-4e63-b387-5f4884f5cb3f" : "Typ",
-    "c3d2100c-8145-4106-af7b-3a31f17fbd29" : "Keine Telefonnummer gefunden.",
-    "d3edd09d-375c-4093-adf5-b64ece35f3e8" : "Keine anzuzeigenden Daten",
-    "c6611e9f-01a7-4565-a520-4d8595952205" : "Meine Details"
+  "dc7dd963-5d2c-47c4-af86-be5f34b09cc6": {
+    "2682aded-a60a-4a78-a59f-9373ae0d9df4": "Meine Details",
+    "18c97a3f-fb78-4bbc-bbd1-2b882fb9f4c9": "E-Mail",
+    "72018e34-61b2-419f-a654-65fe889d8453": "Primäre E-Mail",
+    "7901d093-fca6-4e34-be78-e11fcfb02a2e": "No email address found",
```
Comment (ghost) [inline bundles/http/Citrix/66cf6f1a-e71d-44df-88c5-3db3e99ef7da/i18n/de.json:97]: Why this string is not translated?

File: bundles/http/Citrix/66cf6f1a-e71d-44df-88c5-3db3e99ef7da/i18n/de.json [hunk 2]
```diff
@@ -1,148 +1,148 @@
...
+    "05a95405-b711-4360-849b-5eb0e4cc7aad": "Name",
+    "ccaf5ba6-761a-4186-bb34-0e9126a06839": "Organisationsname",
+    "6ff60f51-b419-4081-a144-ae7737acc990": "Benutzerdetails",
+    "8dcb0ef9-ce83-44a2-94eb-766900e44f28": "Position",
+    "398a12a8-b5a9-4f26-8c4b-4b7d190377f0": "Typ",
+    "d6abfa6c-427b-4141-8b19-edd0b784e066": "E-Mail-Adressen",
+    "285ac3d4-474b-4437-a787-b6af3dda7986": "Zeigen Sie Ihre eigenen Details an und bearbeiten Sie sie.",
+    "db7cc4ca-a6ff-4132-bb8e-bc13886dfcc2": "Ort",
+    "59593fb3-5cd9-4e63-b387-5f4884f5cb3f": "Typ",
+    "c3d2100c-8145-4106-af7b-3a31f17fbd29": "No phone number found",
```
Comment (ghost) [inline bundles/http/Citrix/66cf6f1a-e71d-44df-88c5-3db3e99ef7da/i18n/de.json:112]: Why this string is not translated?

File: bundles/http/Citrix/66cf6f1a-e71d-44df-88c5-3db3e99ef7da/i18n/de.json [hunk 3]
```diff
@@ -1,148 +1,148 @@
...
+    "d3edd09d-375c-4093-adf5-b64ece35f3e8": "User is not part of any organization",
```
Comment (ghost) [inline bundles/http/Citrix/66cf6f1a-e71d-44df-88c5-3db3e99ef7da/i18n/de.json:113]: Why this string is not translated?

File: bundles/http/Citrix/f1c0fcac-0764-4a21-96b8-77a0316e337d/i18n/de.json [hunk 4]
```diff
@@ -1,364 +1,364 @@
...
-    "6d338ca9-cf36-49a8-a61b-07ca91eb05e9" : "Dringlichkeit",
-    "12eeb9f5-2bae-406b-aa45-12ac6667972f" : "Anrufer"
+  "5805e29b-56fb-42c1-944d-867c91adaaca": {
+    "fd5de183-6fc6-4b74-8809-2fb122ef71fb": "Auswirkung",
+    "43e9a327-51ba-4856-85cd-fbae6a2d01fc": "Status",
+    "6ca38495-4465-49f2-a2f8-f99835f6b3b3": "Geöffnet am",
+    "724fb4fb-fd21-426a-ba2f-01a38436cd83": "Details",
+    "58fde4f6-7a95-44f7-af99-bb040638211a": "Der Incident {{number}} '{{short_description}}' wurde Ihnen zugewiesen.",
+    "0515b96f-9e7b-449f-840b-263080c4e597": "Kommentar hinzufügen",
+    "1382a579-1fd6-47cd-ac0b-7d89579b5c46": "Incident {{number}} - '{{short_description}}' has been assigned to {{assigned_to__incident.name}}.",
```
Comment (yiminl-ctx) [inline bundles/http/Citrix/f1c0fcac-0764-4a21-96b8-77a0316e337d/i18n/de.json:47]: the {{assigned_to__incident.name}} has been modified,it should be {{assigned_to_incident.name}}

**Review by ghost — CHANGES_REQUESTED** [review]
Some mustache templates  have been modified, for example:
{{assigned_to__incident.name}} should be {{assigned_to_incident.name}}
{{state__incident.label}} should be {{state_incident.label}}

could you please check all the modified templates?

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
> Some mustache templates have been modified, for example:
> {{assigned_to__incident.name}} should be {{assigned_to_incident.name}}
> {{state__incident.label}} should be {{state_incident.label}}
> 
> could you please check all the modified templates?

Ignore this comment, translation is correct, because the content of the EN template has changed

**Review by yangwa-ctx — APPROVED** [review]

**Comment by sivanatase1-ctxs** [comment]
later changes are merged to the branch

**Pull request closed without merging**